test/specs/reducers/topBlocks.spec.js

```diff
@@ -0,0 +1,14 @@
+import reducer from '../../../src/reducers/topBlocks';
+
+describe('Top Block reducer', () => {
+  describe('When GET_RPC_SERVER action is fired', () => {
```

Comment:
Your description here should be consistent with what you end up naming the action.